--- default.orig	2018-05-18 17:08:45.148657841 +0800
+++ default	2018-05-19 22:25:11.485395033 +0800
@@ -15,13 +15,17 @@
 #
 server {
 	listen 80 default_server;
-	listen [::]:80 default_server;
+	# listen [::]:80 default_server;
 
 	# SSL configuration
 	#
-	# listen 443 ssl default_server;
+	listen 443 ssl default_server;
 	# listen [::]:443 ssl default_server;
 	#
+
+	ssl_certificate /etc/nginx/ssl/nginx.crt;
+	ssl_certificate_key /etc/nginx/ssl/nginx.key;
+
 	# Note: You should disable gzip for SSL traffic.
 	# See: https://bugs.debian.org/773332
 	#
@@ -36,9 +40,9 @@
 	root /var/www/html;
 
 	# Add index.php to the list if you are using PHP
-	index index.html index.htm index.nginx-debian.html;
+	index index.php index.html index.htm index.nginx-debian.html;
 
-	server_name _;
+	server_name AiGO;
 
 	location / {
 		# First attempt to serve request as file, then
@@ -57,6 +61,15 @@
 	#	fastcgi_pass unix:/run/php/php7.0-fpm.sock;
 	#}
 
+	location ~ \.php$ {
+		include snippets/fastcgi-php.conf;
+		fastcgi_pass unix:/run/php/php7.0-fpm.sock;
+
+		# Temporary Solution : 504 Gateway Time-out
+		# limit 2147483 sec = 24 day + 20 hr + 31 min + 23 sec 
+                fastcgi_read_timeout 2147483;
+	}
+
 	# deny access to .htaccess files, if Apache's document root
 	# concurs with nginx's one
 	#
